### DISCUSSION TITLE
Splitting an array into multiple files

### DISCUSSION THREAD

**Discussion by heigum** [TomWright/dasel#429] [Q&A]
I have an xml document with a very large array that I would like to process in Snowflake. However as the parser on the snowflake side is very memory constrained I would like to split the array over multiple files. I feel like I'm missing something obvious but I can't find out how to do this with dasel, which in other respects seem great :) I know I can do this with jq, but it would feel so much cleaner if I could do it using only dasel.

**Comment by TomWright** [comment]
If you can give me some sample input and expected output I'll see what I can do for you
